nixos/modules/programs/looking-glass.nix

```
@@ -0,0 +1,401 @@
+          description = "Allow direct DMA transfers if supported";
+          default = null;
+        };
+        shmFile = mkOption {
+          type = types.nullOr types.path;
+          description = "The path to the shared memory file, or the name of the kvmfr device to use, ie: kvmfr0";
+          default = null;
+        };
+      };
+      win = {
+        title = mkOption {
+          type = types.nullOr types.str;
+          description = "The window title";
+          default = null;
+        };
+        position = mkOption {
+          type = types.nullOr types.str;
+          description = "Initial window position at startup";
+          default = null;
+        };
+        size = mkOption {
+          type = types.nullOr types.str;
+          description = "Initial window size at startup";
+          default = null;
+        };
+        autoResize = mkOption {
+          type = types.nullOr types.bool;
+          description = "Auto resize the window to the guest";
+          default = null;
+        };
+        allowResize = mkOption {
+          type = types.nullOr types.bool;
+          description = "Allow the window to be manually resized";
+          default = null;
+        };
+        keepAspect = mkOption {
+          type = types.nullOr types.bool;
+          description = "Maintain the correct aspect ratio";
+          default = null;
+        };
+        forceAspect = mkOption {
+          type = types.nullOr types.bool;
+          description = "Force the window to maintain the aspect ratio";
+          default = null;
+        };
+        dontUpscale = mkOption {
+          type = types.nullOr types.bool;
+          description = "Never try to upscale the window";
+          default = null;
+        };
+        shrinkOnUpscale = mkOption {
+          type = types.nullOr types.bool;
+          description = "Limit the window dimensions when dontUpscale is enabled";
+          default = null;
+        };
+        borderless = mkOption {
+          type = types.nullOr types.bool;
+          description = "Borderless mode";
+          default = null;
+        };
+        fullScreen = mkOption {
+          type = types.nullOr types.bool;
+          description = "Launch in fullscreen borderless mode";
+          default = null;
+        };
+        maximize = mkOption {
+          type = types.nullOr types.bool;
+          description = "Launch window maximized";
+          default = null;
+        };
+        minimizeOnFocusLoss = mkOption {
+          type = types.nullOr types.bool;
+          description = "Minimize window on focus loss";
+          default = null;
+        };
+        fpsMin = mkOption {
+          type = types.nullOr types.int;
+          description = "Frame rate minimum (0 = disable - not recommended, -1 = auto detect)";
+          default = null;
+        };
+        showFPS = mkOption {
+          type = types.nullOr types.bool;
+          description = "Enable the FPS & UPS display";
```

based on one of the errors,
i suspect the `&` here needs to be escaped for the manual to build
```suggestion
          description = "Enable the FPS &amp; UPS display";
```

---

Thanks that fixed it. There was also a similar problem in the escapeKey description.